index.html

```diff
@@ -1346,26 +1338,25 @@
       </h2>
       <pre class="idl">
         dictionary PaymentDetailsModifier {
-          required sequence&lt;DOMString&gt; supportedMethods;
+          required DOMString supportedMethods;
           PaymentItem total;
           sequence&lt;PaymentItem&gt; additionalDisplayItems;
           object data;
         };
       </pre>
       <p>
         The <a>PaymentDetailsModifier</a> dictionary provides details that
-        modify the <a>PaymentDetailsBase</a> based on <a>payment method
-        identifier</a>. It contains the following fields:
+        modify the <a>PaymentDetailsBase</a> based on a <a>payment method
+        identifier</a>. It contains the following members:
       </p>
       <dl>
         <dt>
           <dfn>supportedMethods</dfn>
```

Comment:
You could use a union `(DOMString or sequence<DOMString>)` and throw if the sequence doesn't have exactly 1 item. Then an array with multiple items can't "accidentally" (maybe intentionally, if the payment provider is a bit evil) end up being accepted as a supported payment identifier.